part-configuration.tex

```diff
@@ -56,17 +56,103 @@ \section{LedgerSMB version numbers}
 
 \subsection{ledgersmb.yaml}
 \label{subsec-global-config-ledgersmb-yaml}
 
+For the default, non debug \texttt{ledgersmb.yaml} \index{ledgersmb.yaml} see  \url{https://github.com/ledgersmb/LedgerSMB/blob/master/doc/conf/ledgersmb.yaml}
 
+\subsubsection{\texttt{cookie}}
+ @@@TODO
+
+\subsubsection{\texttt{db}}
+ @@@TODO
+
+\subsubsection{\texttt{default\_locale}}
+@@@TODO
+
+\subsubsection{\texttt{environment\_variables}}
+ @@@TODO
+
+\subsubsection{\texttt{extra\_middleware}}
+@@@TODO
+
+\subsubsection{\texttt{logging}}
+@@@TODO
+
+\subsubsection{\texttt{login\_settings}}
+@@@TODO
+
+\subsubsection{\texttt{mail}}
+
+Email \index{mail} can be configured by selecting which of the two available transports to use. The default \texttt{ledgersmb.yaml} file contains examples of both.
+
+\begin{description}
+\item{\texttt{Email::Sender::Transport::Sendmail}} – The email is sent using the local server's \texttt{sendmail} \index{sendmail} binary. The options are:
+    \begin{description}
+        \item{\texttt{transport:\$class}} – The required transport to use.
+        \item{\texttt{transport:path}} – optionally provide a path to the directory that contains the 'sendmail' binary.
+    \end{description}
+    
+\item{ LedgerSMB::Mailer::TransportSMTP} - This email \index{email} is sent using a remove SMTP server.  The options are:
+    \begin{description}
+        \item{\texttt{transport:\$class}} – The required transport to use.
+        \item {\texttt{transport:host}} – The required host name of the smtp \index{ SMTP} server.
+        \item {\texttt{transport:port}} – The required port number of the smtp server. Note this might vary depending on whether TLS or SSL is used.
+        \item {\texttt{sasl\_username:\$class}} – The required smtp server authentication method
+        \item {\texttt{sasl\_username:mechanism}} – @@@TODO What options are available here?
+        \item {\texttt{sasl\_username:callback:user}} – The required SMTP user name.  @@@TODO I assume 'the-user' is just a place holder.
+        \item {\texttt{sasl\_username:callback:pass}} – The required SMTP password.  @@@TODO what does 'SECURITY-FIRST' mean?  Is this just a place holder or does it have meaning to pick up the password from some other place?
+    \end{description}
+    
+\end{description}
+
+\subsubsection{\texttt{miscellaneous}}
+@@@TODO
+
+\subsubsection{\texttt{output\_formatter}}
+@@@TODO  I really don't have a good way to format lists explanations.  See below at \texttt{paths:config:workflows} for an example. Guidance welcome.
+
+\subsubsection{\texttt{paths}}
+
+This section configures the various paths used by LedgerSMB. The configuration parameters are:
+
+\begin{description}
+    \item {\texttt{config:locale}} – Path to locale files. Defaults to \texttt{locale/po}.
+    \item {\texttt{config:sql}} – @@@TODO
+    \item {\texttt{config:sql\_data}} – @@@TODO

```

Comment:
Where the reference and masterdata files live (i.e. the files with the names of the countries).

Reply:
Added